lib/analyser/model.py

```diff
@@ -38,15 +38,15 @@ async def add_model_from_dict(cls, dict_data):
         else:
             if await cls.get_model(model.model_name) is None:
                 cls.models.append(model)
-                await cls._update_cb(SUBSCRIPTION_CHANGE_TYPE_ADD, model.model_name, await EnodoModel.to_dict(model))
+                await cls._update_cb(SUBSCRIPTION_CHANGE_TYPE_ADD, await EnodoModel.to_dict(model))
```

Comment:
Is there a very good reason to make the `to_dict` function async? It is currently just returning a `dict` without any async features, and I would actually not expect a "to_dict" function to be async.

Reply:
It once called an other async func. This is indeed no longer the case, I will change it to a normal func